review/internal/service/service_test.go

```diff
@@ -1,88 +1,121 @@
 package service
```

Comment:
Just edited existing tests to use mocks and added few test cases. For production use we should cover more code.